pages/deployments/deploying_to_kubernetes.md.erb

```diff
@@ -0,0 +1,192 @@
+# Deploying to Kubernetes
+
+This tutorial teaches deploying to Kubernetes. You'll need to prepare your
+agents with the required packages, cluster access, and Docker registry access
+before hand.
+
+<%= estimated_time "30 minutes" %>
+
+<%= toc %>
+
+## Overview
+
+<%= image "final_test_pipeline.png" %>
+
+You'll automate deploys from `master` to Kubernetes by the end of the tutorial.
+
+This tutorial uses two pipelines. The first pipeline builds, tests, and pushes
+a Docker image to a registry. The second pipeline-our deploy pipeline-is
+provided a `DOCKER_IMAGE` environmentment for deployment to Kubernetes. You'll
+see how to trigger manual deployments, then link the pipelines for automated
+deployments.
+
+This tutorial assumes:
+
+1. You have an existing test pipeline that pushes a Docker image to a registry
+2. You have a Kubernetes cluster authenticated to the Docker registry
+4. Agents have `kubectl` installed
+3. Agents have `kubectl` access to the Kubernetes cluster
+
+<section class="Docs__note">
+  <p>
+    This setup is outside this tutorial's scope. However, you may refer to the
+    <a href="#notes">notes</a> for high level advice.
```

Comment:
The 'you may refer to' is a bit too aloof, what do you think about something like 'check out the' or 'please see the'?

Reply:
Thanks for this. It helps me get the tone right.